runtime/dart_isolate.cc

```diff
@@ -689,9 +690,14 @@ bool DartIsolate::MarkIsolateRunnable() {
     return false;
   }
 
+  if (Dart_IsError(plugin_registrant_function)) {
```

Comment:
Move this down to where `plugin_registrant_function` is assigned in RunFromLibrary

Reply:
Done.